discovery-provider/src/api/v1/models/playlists.py

```diff
@@ -16,6 +16,7 @@
 playlist_added_timestamp = ns.model(
     "playlist_added_timestamp",
     {
+        "metadata_timestamp": fields.Integer(required=True),
```

Comment:
will this work for legacy tracks, or do we need to may the timestamp to the metadata_timestamp for legacy tracks in this case?